### PR TITLE
Update events + posts

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

**Comment by beatrizserrano** [comment]
@bebatut your event is included here :)

File: _posts/2021-09-14-new-hub.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: "The New Galaxy Community Hub"
+tags: 
+external: https://preview.galaxyproject.org/blog/2021-09-hub2/
```
Comment (bgruening) [inline _posts/2021-09-14-new-hub.md:5]: this is still the preview URL?

File: _posts/2021-09-15-EU-server-cross-disciplinary.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: "UseGalaxy.eu as a cross-disciplinary platform for European researchers"
+tags: report
+external: 
+---
+
+Galaxy is nowadays well-known to be a __cross-discipline platform that connects researchers all over the world__. Many different communities use it and several projects at different levels are supporting the [European Galaxy server](https://usegalaxy.eu/). 
+
+
+At the __German level__, [de.NBI](https://www.denbi.de) provides the cloud infrastructure and the newly established [NFDI](https://www.nfdi.de/) (Nationale Forschungsdateninfrastruktur) covers the data aspects. Within the [DFG](https://www.dfg.de/) (Deutsche Forschungsgemeinschaft) funded project [DataPLANT](https://www.nfdi4plants.de/), scientists will be able to use a well-annotated data management platform for plant research, a key aspect for future reproducibility and reusability of the resources.
```
Comment (bgruening) [inline _posts/2021-09-15-EU-server-cross-disciplinary.md:11]: The different SFBs that we have?

File: _posts/2021-09-14-new-hub.md (hunk 1 above)
Comment (beatrizserrano) [reply _posts/2021-09-14-new-hub.md:5]: oh nein! yeah that's the wrong one

File: _posts/2021-09-15-EU-server-cross-disciplinary.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+In [EOSC-Life](https://www.eosc-life.eu/), the European Galaxy server provides cloud infrastructure and support on the software stack required to implement workflows for the Biomedical Sciences. In the two open calls for training, EOSC-Life has granted a [workshop on Functionally Assembled Terrestrial Ecosystem Simulator ](https://www.eosc-life.eu/news/training-online-galaxy-training-on-functionally-assembled-terrestrial-ecosystem-simulator-fates/) and [Training Infrastructure as a Service (TIaaS)](https://galaxyproject.eu/posts/2021/08/24/tiaas-flyer/). Several EOSC-Life communities ([chemoinformatics](https://www.eosc-life.eu/d1/), [marine genomics](https://www.eosc-life.eu/d4/) and [imaging](https://www.eosc-life.eu/d6/)) have described their use cases with Galaxy workflows. Other EOSC projects have also prominent use cases with Galaxy, like [EOSC-Pillar](https://www.eosc-pillar.eu/), with [reference data](https://www.eosc-pillar.eu/use-cases/exploring-reference-data-through-existing-computing-services-bioinformatics-community) and [EOSC-Nordic](https://www.eosc-nordic.eu/) with the [Climate Science Workbench](https://nordicesmhub.github.io/EOSC/).
```
Comment (bgruening) [inline _posts/2021-09-15-EU-server-cross-disciplinary.md:19]: BY-COVID? CONVERGE?

**Comment by bebatut** [comment]
Thanks @beatrizserrano! I will open a PR to add the link to the Gdoc with meeting details

**Pull request merged**